examples/README.md

```diff
@@ -0,0 +1,39 @@
+### Welcome!
+To successfully execute these examples you'll need to configure some environment variables. To do so:
+
+1. Rename the ".env.template" file located at the root of this folder to ".env"
+1. Uncomment, and assign your Sepolia testnet endpoint to the `RPC_PROVIDER_URL` variable in the ".env" file
```

Comment:
The numbers in the preview look fine (ie go 1,2,3... etc), but the raw .md file seems to all be "1."

Reply:
Yes, that's on purpose. It's a markdown feature so that we don't need to edit all the subsequent numbers if changing something. For me, it's better having '1's than shuffled numbers.
https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Hm, cool, didn't know about this. Thanks